### ISSUE TITLE
Make sure this builds using cabal

### ISSUE THREAD

**Issue by gelisam** [gelisam/hawk#177]
I have switch to stack, but if I publish a new version I should make sure it builds with cabal as well. I guess I could use very narrow bounds, matching the stack lts? That would certainly be less of a maintenance burden than trying every version every time I release. We can always widen them in revisions later.

**Comment by gelisam** [comment]
Turns out the bounds are already correct and that `cabal new-build` works just fine.

I have `cabal new-build` to the Travis configuration (#181), by naively adding `cabal update` and `cabal new-build` commands to the end of the build. It would be better if the two builds were run in parallel, but I'm happy to postpone that complication until later (#182).
